frontend/packages/ceph-storage-plugin/src/components/bucket-class/create-bc.tsx

```diff
@@ -252,12 +252,14 @@ const CreateBucketClass: React.FC<CreateBCProps> = ({ match }) => {
         <BreadCrumbs
           breadcrumbs={[
             {
-              name: _.get(
-                clusterServiceVersion,
-                'spec.displayName',
-                'Openshift Data Foundation Operator',
-              ),
-              path: resourcePathFromModel(ClusterServiceVersionModel, appName, ns),
+              name: _.get(clusterServiceVersion, 'spec.displayName', 'Openshift Data Foundation'),
```

Comment:
here as well, need to confirm with @bipuladh once, but, I dont think we need `_.get`
```suggestion
              name: ODF_MODEL_FLAG ?  'OpenShift Data Foundation' : 'OpenShift Container Storage',
```

Reply:
For ODF just use OpenShift Data Foundation. We don't want to have Operator in the end.

Reply:
@SanjalKatiyar I haven't touched `get` here since that's how things were earlier, and the fact that it already recognises b/w OCS and ODF, so no functionality lost. Just renamed `Openshift Data Foundation Operator` to `Openshift Data Foundation`.

Reply:
ack...

Reply:
Let's use ODF_MODEL_FLAG and standardize the names to OCS Operator or OpenShift Data Foundation.